modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/definition/tree/manager/test/ObjectDefinitionTreeUtilTest.java

```diff
@@ -119,6 +126,123 @@ public void tearDown() {
 		ObjectDefinitionTreeUtil.invalidate();
 	}
 
+	@Test
+	public void testAddNodeObjectEntry() throws Exception {
+
+		// Add node object entry
```

Comment:
Can we not use the term `node`?
Maybe we could use "related" object entry? :thinking: